weblate/vcs/tests/test_vcs.py

```diff
@@ -435,19 +434,406 @@ def add_remote_commit(self, conflict=False, rename=False):
         finally:
             self._class = backup
 
+    def _mock_response(self, json_data, status):
+        """Mock response helper function.
+
+        This function will act as a helper function for response
+        """
+        mock_resp = mock.Mock()
+        mock_resp.status_code = status
+        mock_resp.json = mock.Mock(return_value=json_data)
+        return mock_resp
```

Comment:
It would be less code to use [responses](https://pypi.org/project/responses/) for mocking the requests. We already use it in other places for testing. But it's okay to keep it this way once that was implemented...

Reply:
Okay. Will check that.